lib/internal/fs/promises.js

```diff
@@ -739,6 +747,68 @@ async function readFile(path, options) {
   return PromisePrototypeFinally(readFileHandle(fd, options), fd.close);
 }
 
+async function stream(handle, dest, options) {
+  checkAborted(options.signal);
+  if (!EE) {
+    EE = require('events');
+  }
+  const BUFFER_SIZE = 128 * 2 ** 10;
+  const {
+    start = 0,
+    end,
+  } = options;
```

Comment:
this, and the `checkAborted()` above will fail if `options` is `null` or `undefined`

Reply:
@jasnell The object `options` is [specified](https://github.com/Ayase-252/node/blob/08ed68758631ccfa1e3a213f77c8b8d54b642849/lib/internal/fs/promises.js#L750) as `{}` and validated via [validateObject](https://github.com/Ayase-252/node/blob/08ed68758631ccfa1e3a213f77c8b8d54b642849/lib/internal/validators.js#L143-L156) [function](https://github.com/Ayase-252/node/blob/08ed68758631ccfa1e3a213f77c8b8d54b642849/lib/internal/fs/promises.js#L751) in case of given invalid, so it cannot be null or undefined. Am I missing something?